README.md

```diff
@@ -121,18 +121,40 @@ or create a customized version using the `LogbookBuilder`:
 ```java
 Logbook logbook = Logbook.builder()
     .condition(new CustomCondition())
-    .rawRequestFilter(new CustomRawRequestFilter())
-    .rawResponseFilter(new CustomRawResponseFilter())
     .queryFilter(new CustomQueryFilter())
     .headerFilter(new CustomHeaderFilter())
     .bodyFilter(new CustomBodyFilter())
     .requestFilter(new CustomRequestFilter())
     .responseFilter(new CustomResponseFilter())
-    .formatter(new CustomHttpLogFormatter())
-    .writer(new CustomHttpLogWriter())
+    .sink(new DefaultSink(
+            new CustomHttpLogFormatter(),
+            new CustomHttpLogWriter()
+    ))
     .build();
 ```
 
+### Strategy
+
+Logbook used to have a very rigid strategy how to do request/response logging:
+
+- Requests/responses are logged separately
+- Requests/responses are logged soon as possible
+- Requests/responses are logged as a pair or not logged at all  
+  (i.e. no partial logging of traffic)
+
+Some of those restrictions could be mitigated with custom [`HttpLogWriter`](#writing)
+implementations, but they were never ideal.
+
+Starting with version 2.0 Logbook now comes with a [Strategy pattern](https://en.wikipedia.org/wiki/Strategy_pattern)
+at its core. Make sure you read the documentation of the [`Strategy`](logbook-api/src/main/java/org/zalando/logbook/Strategy.java)
+interface to understand the implications.
+
+Some example implementations can be found here:
+
+- [`BodyOnlyIfErrorStrategy`](logbook-core/src/test/java/org/zalando/logbook/BodyOnlyIfErrorStrategy.java)
+- [`ErrorResponseOnlyStrategy`](logbook-core/src/test/java/org/zalando/logbook/ErrorResponseOnlyStrategy.java)
+- [`WithoutBodyStrategy`](logbook-core/src/test/java/org/zalando/logbook/WithoutBodyStrategy.java)

```

Comment:
What do you think of shipping such example strategies? We know that there is a demand for such behavior and we can easily foresee that users may want to apply this to multiple services. 

* `BodyOnlyIfStatusAtLeastStrategy(int)`
* `StatusAtLeastStrategy(int)`
* `WithoutBodyStrategy`

Reply:
Yeah, probably worth it. I was hoping that the interface is so easy to use that users can just implement it within 2-5 lines.

Reply:
Agreed on the easy-to-be-used, but still gets tedious to put these 5 lines in my 25 micro-services :wink:

Reply:
I created #440 as a follow-up.